common/common.go

```diff
@@ -96,6 +96,8 @@ const (
 	AnnotationKeyHelmHook = "helm.sh/hook"
 	// AnnotationValueHelmHookCRDInstall is a value of crd helm hook
 	AnnotationValueHelmHookCRDInstall = "crd-install"
+	// AnnotationSyncStatusOptions is a comma-separated list of options
+	AnnotationSyncStatusOptions = "argocd.argoproj.io/sync-status-options"
```

Comment:
Not a fan of this key name. What's wrong with compare-options? We use the term compare in other scenarios (e.g. ComparisonError), so it would be consistent with what we already have.

Reply:
Also: `status.comparedTo`

Reply:
we don't use compare in the "core concepts" document, I just felt like it an option around calculating sync status, but I don't feel strongly about this and so I'll change it

Reply:
changed